custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -243,7 +243,12 @@ def updates(self):
                 CASE WHEN last_referral_date BETWEEN %(start_date)s AND %(end_date)s
                 THEN 1 ELSE 0 END
             ) as cases_person_referred
-        FROM "{ucr_tablename}" ucr
+        FROM "{ucr_tablename}" ucr INNER JOIN
```

Comment:
Why is it inner join not left join? Wont it prevent not migrated cases to show up whose entries are not there in migration table.

Reply:
fixed!!